lib/class-wp-rest-menu-items-controller.php

```diff
@@ -97,7 +181,18 @@ public function create_item( $request ) {
 			return new WP_Error( 'rest_post_exists', __( 'Cannot create existing post.', 'gutenberg' ), array( 'status' => 400 ) );
 		}
 
-		$prepared_nav_item = $this->prepare_item_for_database( $request );
+		$prepared_nav_item = $request['prepared_nav_item'];
```

Comment:
```suggestion
		$prepared_nav_item = $this->sanitize( $request );
```
This is a bad pattern to put an object into state like this. Just run the method again.